### PR TITLE
Fix two bugs in autodestroy

### PR DESCRIPTION
**Pull request by pieterdd**
My colleague @kjoep did some bugfixes. I'm filling out the form on his behalf to get it merged into the mainline.

**What kind of change does this PR introduce?** (check at least one)

- [x] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev` branch.
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number) - _It's not related to an existing issue thread as far as we know_
- [x] All tests are passing: https://github.com/vuejs/vue-test-utils/blob/dev/.github/CONTRIBUTING.md#development-setup - _Awaiting build_
- [x] New/updated tests are included - _The TypeScript change is not testable. Anything else the author should cover?_

**Other information:**

### CODE REVIEW

**Comment by Kjoep** [comment]
💪

Op vr 23 okt. 2020 11:28 schreef Pieter De Decker <notifications@github.com

**Comment by afontcu** [comment]
Hi! Thanks for this 🙌  Could you add some unit test to assert that the expected behavior is actually happening?

**Comment by Kjoep** [comment]
@afontcu Added a unit test - this one fails without the fix, but succeeds now.

File: packages/test-utils/dist/vue-test-utils.js [hunk 1]
```diff
@@ -11223,12 +11223,12 @@ function enableAutoDestroy(hook) {
   isEnabled = true;
 
   hook(function () {
-    wrapperInstances.forEach(function (wrapper) {
+    wrapperInstances.forEach((wrapper) => {
```
Comment (lmiller1990) [inline packages/test-utils/dist/vue-test-utils.js]: can you revert this? we only update `dist` when we do a release

**Review by lmiller1990 — COMMENTED** [review]
looks good, just need to revert the dist files - we only update those when we do a release.

File: packages/test-utils/dist/vue-test-utils.js (hunk 1 above)
Comment (pieterdd) [reply packages/test-utils/dist/vue-test-utils.js]: Done, we've reverted the code change.

**Comment by pieterdd** [comment]
We've also removed the second commit since it was obsolete after https://github.com/vuejs/vue-test-utils/pull/1724 got merged: 
image

**Comment by lmiller1990** [comment]
I'll release in the next day or two, thanks!

**Pull request merged**